.readme-partials.yaml

```diff
@@ -227,20 +227,17 @@ custom_content: |
         <artifactId>google-cloud-logging-servlet-initializer</artifactId>
       </dependency>      
       ```
-  #### Population of log entrys' trace/span Id
+  #### Population of Log Entrys' Trace/Span ID
```

Comment:
Sorry -- I just noticed this -- "Log Entrys'" -- that's really weird. The structure is `LogEntry`, which you can't really pluralize or make possessive as Log Entrys' . If you want to pluralize it, you'd have either use the general description, "a log entry", which you could plural-possessive-ize as "log entries' ". But I think I'd just flip this around to say something like "Population of trace/spanID fields in a LogEntry"

Reply:
Done:)